src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentController.kt

```diff
@@ -157,6 +158,35 @@ class DocumentController(
     }
   }
 
+  @PostMapping("/recalls/{recallId}/documents/create")
```

Comment:
Just wondering if we should in due cause remove the create capability from the GET end-points (`/recallNotification` and `/dossier`)?

Reply:
I'd like to do that :) but would mean hooking the create action into the front end somewhere. One to discuss.